src/Sylius/Bundle/GridBundle/Templating/Helper/GridHelper.php

```diff
@@ -16,6 +16,7 @@
 use Sylius\Component\Grid\Definition\Action;
 use Sylius\Component\Grid\Definition\Field;
 use Sylius\Component\Grid\Definition\Filter;
+use Sylius\Component\Grid\Renderer\BulkActionGridRendererInterface;
```

Comment:
not needed?

Reply:
I'll leave it for CS tool, same with license docblocks and other stuff :)